docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
+This can be the complete contributor details, if it is short. 
+Otherwise, it can point to your project documentation which should include
+ a Contributor guide. It also should include your Ansible Matrix chat channels and 
+ [Ansible Community Forum](https://forum.ansible.com/) tags.
+* [Code of Conduct](https://github.com/ansible-community/onboarding/blob/main/CODE-OF-CONDUCT.md) - 
+MUST link to the [Ansible code of conduct](https://docs.ansible.com/ansible/devel/community/code_of_conduct.html). 
+    Do not copy the CoC.
+[DCO](https://github.com/ansible-community/onboarding/blob/main/DCO) - The Developer 
+Certificate of Origin. Do not modify this text.
+* [README.md](https://github.com/ansible-community/onboarding/blob/main/README.md) - 
+A basic introduction to your project, communication channels, and so on.
+It should include your matrix channel if your project has one, and  recommended forum tags,
+with a link to the [Ansible Community Forum](https://forum.ansible.com/). 
+You should not use the README.md for project documentation (user guide, for example),
+ but instead set up a docsite. Even if the docsite is basic, it's a start others can
+contribute to.
+
+## Optional other steps
+While not required, you should consider adding the following to your project repository:
+
+  * Add a [SECURITY.md](https://github.com/ansible-community/onboarding/blob/main/SECURITY.md) 
+  to let a security reporter know how to proceed.
+  * Provide Issue and PR templates to improve the contributor experience.
+
+
+# Creating your community space
+
+Each Ansible ecosystem project should have ways to communicate and collaborate
+ with community members. The following sections cover ways to interact
+ and build your project community.
+
+## Matrix
+Our preferred chat option for Ansible projects is Matrix. Your matrix room name
+should match your project name. You can set up your own matrix room or 
+contact the community team in the forum to be officially added to the Ansible matrix space.
+By doing so your project gets:
+
+- Added to the Space (which functions as the directory)
+- An ansible.com room alias
+- The Admin bot to give you moderation/spam coverage
+- A Matrix meetbot to help you create meeting summaries and logs, automatically posted to the forum
+- (eventually) A room icon based on our template
+
+Contact the community team in the forum to coordinate a new Matrix room for your project.
+
+## Using the Ansible forum
+Ansible introduced the [Ansible Community Forum](https://forum.ansible.com/).
+This is a great place to interact with community members and grow your own project
+community. We have a [set of videos](https://forum.ansible.com/t/discourse-video-demos/102)
+ to help you learn how to use the forum, as well as 
+ [some other tips](https://forum.ansible.com/tag/forum-tips). 
+
+Reach out to the community team to request new tag(s) for your project in the forum.
+
+## Working Groups
```

Comment:
Does Working Group just meet meetings these days (given we talk about Matrix above)

Reply:
No. A working group doesn't have to have a synchronous meeting.